framework/behaviors/SluggableBehavior.php

```diff
@@ -215,7 +217,7 @@ protected function makeUnique($slug)
     /**
      * Checks if given slug value is unique.
      * @param string $slug slug value
-     * @return bool whether slug is unique.
+     * @return boolean whether slug is unique.
```

Comment:
This is wrong change. Should be `bool` because `boolean` can't be used as return type in PHP 7+.